metadata-service/openapi-servlet/src/main/java/io/datahubproject/openapi/entities/EntitiesController.java

```diff
@@ -103,21 +108,28 @@ public ResponseEntity<UrnResponseMap> getEntities(
 
   @PostMapping(value = "/", produces = MediaType.APPLICATION_JSON_VALUE)
   public ResponseEntity<List<String>> postEntities(
-      @RequestBody @Nonnull List<UpsertAspectRequest> aspectRequests) {
+      @RequestBody @Nonnull List<UpsertAspectRequest> aspectRequests,
```

Comment:
Can we instead add this as a header to this endpoint? https://github.com/datahub-project/datahub/blob/master/metadata-service/restli-servlet-impl/src/main/java/com/linkedin/metadata/resources/entity/AspectResource.java#L140

This is the ingestProposal